src/components/Text/Text.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import styles from "./Text.module.scss";
+import { PropTypes } from "react";
+
+const sizeToClass = {
+  h1: styles.text_h1,
+  h2: styles.text_h2,
+  h3: styles.text_h3,
+  h4: styles.text_h4,
+  p: styles.text_paragraph,
+};
+
+export function Text({ text, size = "p" } = {}) {
```

Comment:
prefer `children` to render components content instead of some custom prop

```suggestion
export function Text({ children, size = "p" } = {}) {
```